journal-of-universal-computer-science.csl

```diff
@@ -0,0 +1,529 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" and="text" delimiter-precedes-last="never" demote-non-dropping-particle="never" xmlns="http://purl.org/net/xbiblio/csl">
```

Comment:
not quite sure I understand why you're setting `delimiter-precedes-last="never"` here but then set it to `"always"` [below](https://github.com/citation-style-language/styles/pull/1856/files#diff-553bedbd5da131219a6a3f1fbab7280eR55)
It seems odd this would differ within the style? And if it does, better to set it explicitly each time for clarity.